### PR TITLE
Another item change

### PR DESCRIPTION
**Pull request by pikca-c**
_Project proposals are no longer being accepted as of January 15th, 2023 23:59 EST._

If this a bug fix, documentation update, or other, please describe the purpose of this pull request below:

Sorry for a few last minute changes
The original amazon printer cannot be delivered to where I live before 14 Feb, I have searched different sites and found this one on tmall that arrives before the event starts.

### CODE REVIEW

File: pikca-c.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ With all the components available, I will use the Raspberry Pi (Zero W or Zero 2
 | Product         | Supplier/Link                         | Cost   |
 | --------------- | ------------------------------------- | ------ |
 | Raspberry Pi Zero 2 W   | https://thepihut.com/products/raspberry-pi-zero-2?src=raspberrypi | £17 ($21 USD) |
-| 3.1 inch Graphic OLED Module | https://www.amazon.com/Spacesea-Display-3-12Inch-Controller-500131408A1/dp/B0B6C8RTZY | $23.69 shipping fee |
-| Creality Ender 3 3D Printer | https://www.amazon.com/Comgrow-Creality-Ender-Aluminum-220x220x250mm/dp/B07BR3F9N6 | $199|
+| 3.1 inch Graphic OLED Module | https://www.amazon.com/Spacesea-Display-3-12Inch-Controller-500131408A1/dp/B0B6C8RTZY | $23.69  |
+| Creality Ender 3 3D Printer | https://detail.tmall.com/item.htm?abbucket=20&id=673650763165 | $38.41+$15.89 (Amt over $199 will be paid myself) = $199|
 | 3D modeling software | https://www.sketchup.com/ | Subscription owned |
 | 8 GB SD card | / | Owned |
 | Total           |                                       | $243.69 |
```
Comment (cfanoulis) [inline pikca-c.md:26]: hello! please update your budget with the actual total here (don't forget shipping & taxes!)

**Review by cfanoulis — CHANGES_REQUESTED** [review]
A nit:

File: pikca-c.md (hunk 1 above)
Comment (pikca-c) [reply pikca-c.md]: There was a typo in the price, sorry!

**Review by cfanoulis — APPROVED** [review]
original pr: #206 

great, thanks for that! handing this off to my pardner @hackliv to make sure ops is aware.

**Comment by hackliv** [comment]
Waiting to merge until confirmed this merchant will be used! @pikca-c lmk that the hcb cards don't work on their site

**Review by hackliv — APPROVED** [review]

**Pull request merged**